### DISCUSSION TITLE
Proposal: Allow loading more comments in either direction

### DISCUSSION THREAD

**Discussion by iCodeSometime** [community/community#10386] [Projects and Issues]
When an issue contains a large amount of comments, the comments in the middle are collapsed with a "load more" button.
When pressed, the button loads additional comments, starting from the first comments missing from the section.

Sometimes - especially when there are many comments - I really only want additional context from comments shown below the missing portion.

Adding the ability to load additional comments from the bottom up would greatly improve responsiveness for this common use case, as well as reduce load on your servers by making it easier to load less and still get the content I'm looking for

**Comment by garry-ut99** [comment]
Duplicate of: ["Load more comments" in issues and PRs needs to work better](https://github.com/orgs/community/discussions/5119)
